subtitle_downloader.py

```diff
@@ -0,0 +1,88 @@
+#-------------------------------------------------------------------------------
+# Name      : subtitle downloader
+# Purpose   : One step subtitle download
+#
+# Authors   : manoj m j, arun shivaram p, Valentin Vetter, niroyb
+# Edited by : Valentin Vetter
+# Created   :
+# Copyright : (c) www.manojmj.com
+# Licence   : GPL v3
+#-------------------------------------------------------------------------------
+
+# TODO: use another DB if subs are not found on subDB
+
+import hashlib
+import os
+import sys
+
+PY_VERSION = sys.version_info[0]
+if PY_VERSION == 2:
+    import urllib2
+if PY_VERSION == 3:
+    import urllib.request
+
+
+def get_hash(file_path):
+    read_size = 64 * 1024
+    with open(file_path, 'rb') as f:
+        data = f.read(read_size)
+        f.seek(-read_size, os.SEEK_END)
+        data += f.read(read_size)
+    return hashlib.md5(data).hexdigest()
+
+def get_subtitles(file_hash):
+    headers = {'User-Agent': 'SubDB/1.0 (subtitle_downloader/1.0; http://github.com/manojmj92/subtitle_downloader)'}
+    url = "http://api.thesubdb.com/?action=download&hash=" + file_hash + "&language=en"
+    if PY_VERSION == 3:
+        req = urllib.request.Request(url, None, headers)
+        response = urllib.request.urlopen(req).read()
+    if PY_VERSION == 2:
+        req = urllib2.Request(url, '', headers)
+        response = urllib2.urlopen(req).read()
+    return response
+
+def is_movie_file_extension(extension):
+    return extension in [".avi", ".mp4", ".mkv", ".mpg", ".mpeg", ".mov", ".rm", ".vob", ".wmv", ".flv", ".3gp"]
+
+def sub_download(file_path):
+    # Skip this file if it is not a video
+    root, extension = os.path.splitext(file_path)
+    if not is_movie_file_extension(extension):
+        return
+
+    if not os.path.exists(root + ".srt"):
```

Comment:
You could move this condition to the early return above: 
if os.path.exists(root + ".srt") or not is_movie_file_extension(extension)